security/src/main/java/no/difi/meldingsutveksling/IntegrasjonspunktNokkel.java

```diff
@@ -2,6 +2,8 @@
 
 import no.difi.asic.SignatureHelper;
 import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import no.difi.meldingsutveksling.config.KeyStoreProperties;
+import no.difi.meldingsutveksling.lang.KeystoreProviderException;
 import org.springframework.core.io.Resource;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.springframework.core.io.Resource'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)